src/components/chat/chat-utils.ts

```
@@ -0,0 +1,154 @@
+import { type TFunction } from 'i18next';
+
+import { getAvatarUrl } from '@/lib/utils';
+import { ChatChannelType, type ChatChannelResultData, type ChatGifMetadata, type ChatImageMetadata, type ChatLocationMetadata } from '@/models/v4/chat';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)